impl/maven-cli/src/main/java/org/apache/maven/cling/extensions/ExtensionConfigurationModule.java

```diff
@@ -36,6 +36,7 @@ public class ExtensionConfigurationModule implements Module {
 
     private final CoreExtensionEntry extension;
     private final UnaryOperator<String> callback;
+
     private final DefaultInterpolator interpolator = new DefaultInterpolator();
```

Comment:
What about this one?

Reply:
This component is loaded very early without the full container afaik.  This would require an `Interpolator` to be bound to the `ProtoSession` and `ProtoLookup`.  Currently the only component added is `ClassWorld`.